scripts/apps/contacts/components/ContactHeader.jsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {JobTitle} from 'apps/contacts/components/fields';
```

Comment:
I think it's better to use relative imports if it's part of same package, like from `./fields` here, but not a big deal